### PR TITLE
feat(wich-key): binding to save without formatting

### PR DESCRIPTION
**Pull request by danielo515**
# Description

summary of the change

A lot of people is concerned when they save a file and they don't want auto-format to be applied to it. This adds a nice which key mapping to do that.

## How Has This Been Tested?

This is a command I execute often, there is nothing fancy about it

### CODE REVIEW

**Comment by danielo515** [comment]
@kylo252 anything against?

**Comment by danielo515** [comment]
Sorry for bumping this, but I think it is something will help people.

**Comment by danielo515** [comment]
@ChristianChiarulli would you mind having a look?

**Comment by LostNeophyte** [comment]
I've added that keymapping to the default config (not yet merged) and even my own, it's quite useful, thanks :)
I'm not sure about adding it to lvim because formatting on save is disabled by default

PS: set the base branch to master

**Comment by danielo515** [comment]
> I've added that keymapping to the default config (not yet merged) and even my own, it's quite useful, thanks :) I'm not sure about adding it to lvim because formatting on save is disabled by default
> 
> PS: set the base branch to master

Already changed the base to master.
The keybinding is also useful even if you don't use autoformat, because it just disables all autocomands when saving, so you may have a git auto commit autocommand, a copy file autocommand, and you just want to save without running them. What about changing the name conditionally based in the presence of autoformat or not?

**Comment by LostNeophyte** [comment]
I guess you can change the keybind description to `Save without autocommands`, although beginners may not know that it'll work for formatting then, @kylo252 what do you think?

File: lua/lvim/core/which-key.lua [hunk 1]
```diff
@@ -92,6 +92,7 @@ M.config = function()
         f = { "<cmd>Telescope buffers<cr>", "Find" },
         b = { "<cmd>BufferLineCyclePrev<cr>", "Previous" },
         n = { "<cmd>BufferLineCycleNext<cr>", "Next" },
+        W = { "<cmd>noautocmd w<cr>", "Save without formatting" },
```
Comment (kylo252) [inline lua/lvim/core/which-key.lua:95]:
```suggestion
        W = { "<cmd>noautocmd w<cr>", "Save without formatting (noautocmd)" },
```

**Review by kylo252 — APPROVED** [review]

**Pull request merged**